php/commands/role.php

```diff
@@ -367,6 +383,167 @@ private static function persistence_check() {
 		if ( !$wp_roles->use_db )
 			WP_CLI::error( "Role definitions are not persistent." );
 	}
+
+	private static function get_role_capability( $role_key ) {
```

Comment:
Why did you need to create this method? Seems like a lot of code to internalize — not sure if it's a great idea.